### PR TITLE
Upgrade Mixlib-ShellOut -> 3

### LINKED ISSUE #1: Gem homepage metadata incorrect
The homepage listed in the gem metadata is not what the repo is named.

### PR DESCRIPTION
**Pull request by kvivek1115**
- Minor gem spec changes.
 - Update the homepage meta link.

Fixes: https://github.com/adejoux/aix-wpar/issues/1, https://github.com/adejoux/aix-wpar/issues/2

Signed-off-by: Vivek Singh <vivek.singh@msystechnologies.com>

### CODE REVIEW

**Comment by kvivek1115** [comment]
@adejoux hope you are around any chance to look into this PR?

**Comment by kvivek1115** [comment]
@adejoux Any chance to look into this PR?
eagerly waiting for your response  :cry:

File: aix-wpar.gemspec [hunk 1]
```diff
@@ -1,24 +1,18 @@
...
+  gem.files         = %w{LICENSE} + Dir.glob("*.gemspec") + Dir.glob("{lib,examples}/**/*")
   gem.test_files    = gem.files.grep(%r{^(test|spec|features)/})
   gem.name          = "aix-wpar"
   gem.require_paths = ["lib"]
   gem.version       = WPAR::VERSION
 
-  deps.each do |dep, constraints|
-    gem.add_runtime_dependency dep, *constraints
-  end
+  gem.add_dependency "mixlib-shellout", "~> 3"
```
Comment (tas50) [inline aix-wpar.gemspec]: we should make this > 2 < 4 so we maintain compat with older chef-client releases that require 2

**Comment by kvivek1115** [comment]
Closing this PR seems repo is not in an active state.

**Pull request closed without merging**